Formula/rtags.rb

```diff
@@ -28,6 +28,45 @@ def install
     end
   end
 
+  def caveats; <<-EOS.undent
+    The instance of rtags that run via brew services listens to
+    the socket file at #{var}/.rdm
+
+      rc --socket-file=#{var}/.rdm ...
+    EOS
+  end
+
+  plist_options :startup => false
```

Comment:
Ah, I see. I got `plist_options :startup => false` from the formula for `kwm`.
For this plist configuration, we can just leave it out, since `launchd` listens to the socket at `~/.rdm` for activity and launches `rdm` automatically.